pkg/driver/controller.go

```diff
@@ -58,6 +58,17 @@ var (
 
 const isManagedByDriver = "true"
 
+// BlockSizeFSTypes contains the filesystems that a custom block size is supported on
+var (
+	BlockSizeFSTypes = map[string]struct{}{
```

Comment:
Why is this exported? Suggest `s/BlockSizeFSTypes/blockSizeFSTypes/`.

But really since this list is exactly `ValidFSTypes`, why is it being added at all?

Reply:
oops, artifact of copy-pasting `ValidFSTypes`, fixed

Reply:
> But really since this list is exactly `ValidFSTypes`, why is it being added at all?

Care to comment on this? My concern is that this is a bug-in-waiting. Some day, someone is going to add a new entry to  `ValidFSTypes` and forget to add it here. So the benefit it's providing now should outweigh the cost of that future bug.

Reply:
Oh, that's actually a typo on my part, this list should _not_ contain NTFS, as we don't (currently) support custom block sizes on that filesystem. To be more general: the difference is that `ValidFSTypes` contains _all_ possible fstypes, and this array contains the fstypes that we support custom block sizes for.

Reply:
In that case, I suggest making it a negative list instead of a positive one. That is, we can safely assume that we can specify a block size for a file system unless it is in a list of known defectives.

Rationale:
* The worst-case outcome of a "supported" list is the future bug I described above. This list will have to be updated whenever a new file system type is added.
* The worst-case outcome of a "not supported" list is that some customer will try to set the block size and it will either be ignored or will cause the FS creation to fail. This list will likely never be touched again.

I think the second outcome is the lesser of two evils.

Reply:
Moved to exclusion-based list, and now check in both the controller and node (to account for when the controller allows through an invalid fstype)